app/services/iaa_reporting_helper.rb

```diff
@@ -37,4 +46,27 @@ def iaas
         end.compact
       end.sort_by(&:key)
   end
+
+  def partner_accounts
+    Agreements::PartnerAccount.
+      includes(integrations: :service_provider).
+      flat_map do |partner_account|
+        puts"partner_account: #{partner_account.inspect}"
```

Comment:
can you please remove this

Reply:
I thought i removed all. I am double checking.